pdfarranger-appdata.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/data/com.github.jeromerobert.pdfarranger.metainfo.xml b/data/com.github.jeromerobert.pdfarranger.metainfo.xml
+index e62c1ef..163de96 100644
+--- a/data/com.github.jeromerobert.pdfarranger.metainfo.xml
++++ b/data/com.github.jeromerobert.pdfarranger.metainfo.xml
+@@ -23,7 +23,7 @@ See file COPYING or go to <http://www.gnu.org/licenses/> for full license detail
+   <url type="homepage">https://github.com/pdfarranger/pdfarranger</url>
+   <screenshots>
+     <screenshot type="default">
+-      <image>https://github.com/pdfarranger/pdfarranger/raw/master/data/screenshot.png</image>
++      <image>https://github.com/pdfarranger/pdfarranger/raw/1.7.0/data/screenshot.png</image>
```

Comment:
Argh, I missed that when renaming master to main :/ Thanks for the hint, will fix upstream asap, but it will probably need another release to fix this in the consumers.

Reply:
My tools to build flatpak packages always checks on the appdata, so they caught this. The build would have failed on flathub.

BTW pining on the tag is to ensure the right version is used.

Reply:
Turns out I did fix upstream, but - of course - the 1.7.0 snapshot is from before that fix. Shouldn't be a problem when the next release comes out. Pinning to the tag is of course ideal, but I'll stick with main for the stuff I maintain. It leaves more room for human error than it's worth imo (unless it's automated, that is).